float32[] offset

float32 targetYaw

float32 yawThreshold
float32 posThreshold

---

int8 ack